### DISCUSSION TITLE
Welcome to SchemaCrawler Discussions!

### DISCUSSION THREAD

**Discussion by schemacrawler** [schemacrawler/SchemaCrawler#392] [General]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about
  * Share ideas
  * Engage with other community members
  * Welcome others and are open-minded. Remember that this is a community we
  build together

**Comment by adriens** [comment]
Hello there, seems like Github want to shoot a silver bullet on gitter 🤣
